### ISSUE TITLE
Bug: URL path join is not correct

### ISSUE THREAD

**Issue by adrianmg** [oven-sh/bun#474]
👋  I'm sharing this after [some discussion on Discord](https://discord.com/channels/876711213126520882/887787428973281300/995203100220596305).

<details>
    <summary>📦 package.json</summary>

```package.json
"dependencies": {
  "@prisma/client": "^4.0.0",
  "@radix-ui/react-dialog": "^0.1.8-rc.50",
  "@radix-ui/react-popover": "^0.1.7-rc.43",
  "@radix-ui/react-radio-group": "^0.1.6-rc.46",
  "bun-framework-next": "^12.1.5",
  "contentlayer": "^0.2.6",
  "next": "^12.2.1",
  "next-contentlayer": "^0.2.6",
  "npm": "^8.13.2",
  "react": "^18.2.0",
  "react-dom": "^18.2.0",
  "swr": "^1.3.0"
},
"devDependencies": {
  "@types/react": "^18.0.15",
  "autoprefixer": "^10.4.7",
  "eslint": "^8.19.0",
  "eslint-config-next": "^12.2.1",
  "eslint-plugin-tsdoc": "^0.2.16",
  "postcss": "^8.4.13",
  "prisma": "^4.0.0",
  "stylelint-config-recommended": "^8.0.0",
  "tailwindcss": "^3.1.4",
  "typescript": "^4.7.4"
}
```

</details>

When running `bun bun` I get this error:

```
warn: Could not resolve: ".prisma/client/index". Maybe you need to "bun install"?

  ...require('.prisma/client/index'),
             ^
/Users/adrianmg/Projects/my-project/node_modules/@prisma/client/index.js:2:14 32
```

<img width="899" alt="image" src="https://user-images.githubusercontent.com/589285/178093776-3e1d21c0-4dca-4663-adc8-08db995f0851.png">

**Comment by birkskyum** [comment]
Revision: 0.8.2+4b63ced72dc1b304e737d9ea055c8b6e75f46119


Repro:

```ts
const {PrismaClient} = require("@prisma/client")
```

$ `bun build index.ts`

Result:

```sh
➜ bun build index.ts  


error: Could not resolve: ".prisma/client/index-browser". Maybe you need to "bun install"?
const prisma = require('.prisma/client/index-browser')
                       ^
/Users/admin/repos/bun-kitchensink/node_modules/@prisma/client/index-browser.js:1:24 23
```

**Comment by Jarred-Sumner** [comment]
Prisma inserts a `.prisma` directory

I don't think this is a URL path joining bug. I think it's just missing postinstall.

**Comment by birkskyum** [comment]
- #606

**Comment by birkskyum** [comment]
Yep, adding it to trustedDependencies, and deleting bun.lockd + node_modules and running `bun install` again did the trick.

**Comment by Electroid** [comment]
Duplicate of #4959

**Comment by Jarred-Sumner** [comment]
Fixed by @dylan-conway and @paperdave in #7132

This will be part of the Bun v1.0.17 release, which ships in a few hours from the time of writing.
